sql_grader/mixins/scenario.py

```diff
@@ -62,7 +63,8 @@ def workbench_scenarios(cls):
         """
         module = cls.__module__
         module = module.split('.', maxsplit=1)[0]
-        directory = pkg_resources.resource_filename(module, 'scenarios')
-        files = _find_files(directory)
+        module_ref = importlib.import_module(module)
```

Comment:
Please follow this:
https://github.com/openedx/xblock-submit-and-compare/pull/187/files#diff-ac87fb3db8e79e2e47bcffacecc595d0a17869b5f71c02eb2c38298f200663efR5